lib/commands/command-info.bash

```diff
@@ -2,9 +2,21 @@
 # shellcheck source=lib/functions/plugins.bash
 . "$(dirname "$(dirname "$0")")/lib/functions/plugins.bash"
 
+get_shell() {
+  if [[ -z $FISH_VERSION ]]; then
+    echo "fish, version $FISH_VERSION"
+  elif [[ -z $ZSH_VERSION ]]; then
+    echo "zsh, version $ZSH_VERSION"
+  elif [[ -z $BASH_VERSION ]]; then
+    echo "bash, $BASH_VERSION"
+  else
+    echo $($SHELL --version)
```

Comment:
I believe this line can be simplified to `$SHELL --version` since the shell will print the version anyway.

Reply:
like I had previously?

Reply:
Previously this line was:

```
echo $SHELL --version
```

Now it is (and it behaves correctly):

```
echo $($SHELL --version)
```

But it can be simplified to just:

```
$SHELL --version
```

Reply:
gotcha, that is clearer! thanks.